_schema.graphql

```diff
@@ -2696,1341 +2710,187 @@ type BuyOrder implements Order {
+    before: String
+    last: Int
+  ): ShowConnection
+  fairs(
+    sort: FairSorts
+    status: EventStatus
+    after: String
+    first: Int
+    before: String
+    last: Int
+  ): FairConnection
+  sponsoredContent: CitySponsoredContent
 }
 
-type CommerceShip {
-  addressLine1: String
-  addressLine2: String
-  city: String
-  country: String
-  name: String
-  phoneNumber: String
-  postalCode: String
-  region: String
+type CitySponsoredContent {
+  introText: String
+  artGuideUrl: String
+  featuredShows: [Show]
+  shows(
+    sort: PartnerShowSorts
+    status: EventStatus
+    after: String
+    first: Int
+    before: String
+    last: Int
+  ): ShowConnection
 }
 
-# Shipping information
-input CommerceShippingAttributes {
-  addressLine1: String
-  addressLine2: String
-  city: String
-  country: String
-  name: String
-  phoneNumber: String!
-  postalCode: String
-  region: String
-}
+type Collection implements Node {
+  # A globally unique ID.
+  __id: ID!
 
-# Autogenerated input type of SubmitOrder
-input CommerceSubmitOrderInput {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
+  # A type-specific ID.
   id: ID!
-}
-
-# Autogenerated return type of SubmitOrder
-type CommerceSubmitOrderPayload {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
 
-  # A union of success/failure
-  orderOrError: CommerceOrderOrFailureUnion!
+  # A type-specific Gravity Mongo Document ID.
+  _id: ID!
+  cached: Int
+  artworks_connection(
+    after: String
+    first: Int
+    before: String
+    last: Int
+    private: Boolean = false
+    sort: CollectionSorts
+  ): ArtworkConnection
+  description: String!
+  default: Boolean!
+  name: String!
+  private: Boolean!
+  slug: String!
 }
 
-# Autogenerated input type of SubmitOrderWithOffer
-input CommerceSubmitOrderWithOfferInput {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
-  offerId: ID!
+enum CollectionSorts {
+  POSITION_ASC
+  POSITION_DESC
 }
 
-# Autogenerated return type of SubmitOrderWithOffer
-type CommerceSubmitOrderWithOfferPayload {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
+type CollectorProfileType {
+  # A globally unique ID.
+  __id: ID!
 
-  # A union of success/failure
-  orderOrError: CommerceOrderOrFailureUnion!
-}
+  # A type-specific ID.
+  id: ID!
+  email: String
+  name: String
+  confirmed_buyer_at(
+    # This arg is deprecated, use timezone instead
+    convert_to_utc: Boolean
+    format: String
 
-# Autogenerated input type of SubmitPendingOffer
-input CommerceSubmitPendingOfferInput {
-  # A unique identifier for the client performing the mutation.
-  clientMutationId: String
-  offerId: ID!
-}
+    # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
+    timezone: String
+  ): String
+  collector_level: Int
+  self_reported_purchases: String
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 2868;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `self_reported_purchases`